src/components/Modals/RulesModal.vue

```diff
@@ -116,25 +116,130 @@
                     </li>
                     <li><h5>Hack</h5>
                       <div>
-                        <img src="../../static/cardImg/H.png" alt="Instruction 1 Card" class="cardImage">
+                        <img src="../../../static/cardImg/H.png" alt="Instruction 1 Card" class="cardImage">
                       </div>
 
                       <div>
                         <ol>
                           <li>Hack cards let you attack another player's program.</li>
                           <li>When you play a Hack card, you choose which stack of cards from another player's playing
                             fields will be destroyed. </li>
-                          <li>You cannot hack a Group card.</li>
+                          <li>You cannot hack a Group card or someone with a Firewall card active.</li>
                         </ol>
                         <b>Relation to computer programming:</b>
-                        Sometimes a mean or mischievous computer programmer. known as a <i>hacker</i> will change a computer program
+                        Sometimes a mean or mischievous computer programmer. Known as a <i>hacker</i> will change a computer program
```

Comment:
The period should have been removed instead. Otherwise there is a sentence fragment. Check the other descriptions for this as well.